updated support page (#2560)
SoftwareAG/c8y-docs: content/additional-resources/contacting-support.md

  - getting_started
---

Product support for {{< product-c8y-iot >}} is provided to licensed customers via the [{{< c8y-support-portal >}}]({{< c8y-support-link >}}). The services on this portal require that you have an account. If you do not yet have one, you can send a request by email to [{{< c8y-support-email >}}]({{< c8y-support-email >}}).

> Once you have an account, and depending on role-associated permissions, you can:

Author: ```suggestion
Once you have an account, and depending on role-associated permissions, you have the following options:
```

> - Download products.

Author: ```suggestion
- ** Support**: Access technical assistance, documentation, and submit support tickets.
- **Feedback**: Share your thoughts and suggestions to help improve features and user experience.
- **Managed Services**: Access premium platform management, monitoring solutions, and specialized support.
```

> - Search the Knowledge Base for articles.

Author: ```suggestion
```

> - Open and update support tickets.

Author: ```suggestion
```

> - Share product feedback.

Author: ```suggestion
```
